Module not found error when updating library through npm

### Bug description

If I update my solid client library(from 1.10.0 to the latest one which is 1.11.0) through npm webpack stops working and gives a "Module not found" error:
```
ERROR in ./src/index.js 3:0-85
Module not found: Error: Can't resolve '@inrupt/solid-client/access/universal' in '/Applications/MAMP/htdocs/soliddashboard/src'
resolve '@inrupt/solid-client/access/universal' in '/Applications/MAMP/htdocs/soliddashboard/src'
  Parsed request is a module
  using description file: /Applications/MAMP/htdocs/soliddashboard/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Applications/MAMP/htdocs/soliddashboard/src/node_modules doesn't exist or is not a directory
      looking for modules in /Applications/MAMP/htdocs/soliddashboard/node_modules
        existing directory /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client
          using description file: /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/package.json (relative path: .)
            using exports field: ./dist/access/universal.mjs
              using description file: /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/package.json (relative path: ./dist/access/universal.mjs)
                no extension
                  Field 'browser' doesn't contain a valid alias configuration
                  /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/dist/access/universal.mjs doesn't exist
                .js
                  Field 'browser' doesn't contain a valid alias configuration
                  /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/dist/access/universal.mjs.js doesn't exist
                .json
                  Field 'browser' doesn't contain a valid alias configuration
                  /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/dist/access/universal.mjs.json doesn't exist
                .wasm
                  Field 'browser' doesn't contain a valid alias configuration
                  /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/dist/access/universal.mjs.wasm doesn't exist
                as directory
                  /Applications/MAMP/htdocs/soliddashboard/node_modules/@inrupt/solid-client/dist/access/universal.mjs doesn't exist
      /Applications/MAMP/htdocs/node_modules doesn't exist or is not a directory
      /Applications/MAMP/node_modules doesn't exist or is not a directory
      /Applications/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory

webpack 5.51.1 compiled with 1 error in 1596 ms
ℹ ｢wdm｣: Failed to compile.
```


### To Reproduce

1. Install 1.10.0
2. Update to 1.11.0
3. start webpack

### Environment

```
% npx envinfo --system --npmPackages --binaries --npmGlobalPackages --browsers

  System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
    Memory: 2.10 GB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.5.0 - ~/.nvm/versions/node/v16.5.0/bin/node
    npm: 7.19.1 - ~/.nvm/versions/node/v16.5.0/bin/npm
  Browsers:
    Firefox: 91.0.2
    Safari: 14.1.1
  npmPackages:
    @inrupt/solid-client: ^1.11.1-fix1226-null-getdate-1169648447-4070-1629965841.0 => 1.11.1-fix1226-null-getdate-1169648447-4070-1629965841.0 
    @inrupt/solid-client-authn-browser: ^1.9.1 => 1.11.2 
    @inrupt/vocab-common-rdf: ^0.7.5 => 0.7.5 
    buffer: ^6.0.3 => 6.0.3 
    css-loader: ^6.1.0 => 6.2.0 
    style-loader: ^3.1.0 => 3.2.1 
    uuid: ^8.3.2 => 8.3.2 
    webpack: ^5.45.1 => 5.51.1 
    webpack-cli: ^4.7.2 => 4.8.0 
    webpack-dev-server: ^3.11.2 => 3.11.2 
  npmGlobalPackages:
    browserify: 17.0.0
    npm: 7.19.1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@ The following changes have been implemented but not released yet:
 
 - In some cases, Thing.getDate() would return null while Thing.setDate() had been
   called prior. Thanks to a contribution from @AJamesPhillips, this is now fixed.
+- The submodule export for `./access/universal` was broken.
 
 The following sections document changes that have been released already:
 

--- a/src/index.ts
+++ b/src/index.ts
@@ -340,4 +340,4 @@ export * as access_v1 from "./access/universal_v1";
  * functions directly from @inrupt/solid-client/access/universal.
  * @deprecated Please import directly from the "access/universal" module.
  */
-export * as access from "./access/universal_v1";
+export * as access from "./access/universal";

